tests/training/test_model_trainer.py

```
@@ -327,6 +327,15 @@ def test_trainer_torch_dataset(config, tmp_path: str):
     model_trainer = ModelTrainer(config, data_pipeline_fw="torch_dataset")
     assert model_trainer.dir_path == "."
 
+    ##### test for reusing np chunks path
+    with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)